src/Assertion/Validator/Pipes/CheckPublicKeySignature.php

```diff
@@ -119,7 +119,7 @@ protected function validateWithOpenSsl(string $signature, string $verifiable, We
     {
         if (! $publicKey = openssl_pkey_get_public($credential->public_key)) {
             throw AssertionException::make('Public key is invalid: '.openssl_error_string());
-        }
+        }https://
```

Comment:
I think this was left by mistake.

Reply:
I really have to disabling paste with middle click.